include/magma/vk/vulkan_common.h

```diff
@@ -1,5 +1,5 @@
 #pragma once
-#include <vulkan/vulkan.h>
+#include <vulkan/vulkan.hpp>
 #include <stdexcept>
 #include <vector>
 #include "magma/app/log.hpp"
```

Comment:
may add \n between standard lib and custom lib

Reply:
no actual need for this. There only one include from our lib

Reply:
It creates some order.
Usually

1. Custom h file which contain class definition 
2. Standard lib 
3. Custom lib